src/bptl/work_units/zgw/tasks/zaak_relations.py

```diff
@@ -310,6 +311,44 @@ def perform(self) -> Optional[dict]:
             request_headers=headers,
         )
 
+        try:
+            bijdrage_aard_omgekeerde_richting = check_variable(
+                variables, "bijdrageAardOmgekeerdeRichting", empty_allowed=True
```

Comment:
Could you add `bijdrageAardOmgekeerdeRichting` to the docstring of the task? (in the "optional process variables" section)